lib/violations_testing.rb

```diff
@@ -0,0 +1,5 @@
+class Foo
+  def bar
+    "A very long string that is going to be flagged by Rubocop for being too long for its standards. But oh well :)"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.